phoenicis-repository/src/main/java/org/phoenicis/repository/dto/CategoryDTO.java

```diff
@@ -92,9 +102,19 @@ public String getName() {
         return (o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName());
     }
 
+    @Override
+    public CategoryDTO translate() {
+        List<ApplicationDTO> applicationDTOS = new ArrayList();
```

Comment:
We have it like this in several places (IntelliJ IDEA creates it like this). Unless we introduce a rule and have it the same way everywhere, I don't see a reason to change it here.